src/components/Card/Card.styles.ts

```diff
@@ -0,0 +1,109 @@
+import styled from 'styled-components';
+import { EStatuses } from './Card.types';
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+export const CardContainer = styled.div`
+  display: flex;
+  justify-content: space-around;
+  flex-direction: column;
+  padding: 5px 10px;
+  background-color: ${({ theme }) => theme.colors.primary};
+  gap: 5px;
+  border-radius: 10px;
+`;
+
+export const CardBadgeContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 3px;
+`;
+
+export const CardBadge = styled.div`
+  background-color: ${(props) => props.color || props.theme.colors.grey};
+  display: flex;
+  align-items: center;
+  flex-basis: 25%;
+  text-align: center;
+  justify-content: center;
+  border-radius: 5px;
+`;
+
+export const CardBadgeText = styled.p`
```

Comment:
It would be very nice to introduce `Text` component that have a few props like `size`/`weight`/`color` and etc and use it everywhere instead of repeating the same styles many times.